example/.mage/tools.mk

```diff
@@ -2,16 +2,30 @@ mage_folder := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 mage_generated_path := $(mage_folder)/gen
 mage_targets_file := $(mage_generated_path)/targets.mk
 mage := $(mage_generated_path)/local-mage
+mgmake := $(mage_folder)/mgmake.go
+
+define mgmake_content
+// +build mage
+
+package main
+
+// mage:import
+import _ "go.einride.tech/mage-tools/mgmake"
+endef
+
+export mgmake_content
```

Comment:
why is this needed?

Reply:
From what i understood while reading the Makefile documentaiton, it was needed to make `mgmake_content` a usable variable

Reply:
ah! is that specific to `define`? (never used that before)

Reply:
Yeah, first time for me too. I gave it a try without the export, and it seems to be working fine without it :)

Reply:
woppa!